harmonize console log background color

SalmaanShire365 opened the pull request:
Fixes Issue #654

Changes:

Updated Console Log background color in Shuffler Tab to match A.I. Tab styling
Updated Console Log and Flight Log background color in Read Brain Tab to match A.I. Tab styling
Applied consistent darker gray shade across all console output areas for improved visual harmony

Before: Console logs in Shuffler and Read Brain tabs had lighter background colors
After: All console log areas now share the same darker gray background as the A.I. Tab
Testing:
<img width="1920" height="1061" alt="2025-12-09-204742_1920x1061_scrot" src="https://github.com/user-attachments/assets/6bf5e41a-178d-4958-8399-722d64fa6352" />
<img width="1918" height="1062" alt="2025-12-09-211952_1918x1062_scrot" src="https://github.com/user-attachments/assets/2e6d638d-f997-4f73-ac33-525ac26e2dcf" />


 Verified color uniformity across Shuffler, Read Brain, and A.I. tabs
 Confirmed text readability was maintained with the new background color

SalmaanShire365 commented:
TIcket solved - pull request pending #655

3C-SCSU reviewed (approved):
@SalmaanShire365 
Feedback >> (1) > {2: +66/-39} [ The source code meets all requirements stipulated by the open-source development [code of ethics ](https://github.com/3C-SCSU/Avatar/wiki/A-Guide-to-Contributor-Compliance)policies; Some conflicts needs to be fixed first; Can't be merge it in the current form. ]

SalmaanShire365 commented:
Merged upstream changes from main branch:

**File Reorganization:**
- Renamed `BrainwaveReading.qml` → `ReadBrain.qml`
- Renamed `brainwave_reading_tab.py` → `read_brain_tab.py`

**New API Modules Created:**
- `cloud_api.py` - Cloud functionality backend (143 lines)
- `developers_api.py` - Developer tools API (97 lines)
- `shuffler_api.py` - File shuffler functionality backend (83 lines)

**Major Refactoring:**
- `GUI5.py` significantly reduced (~300 lines removed)
  - File shuffler logic extracted to `shuffler_api.py`
  - Following clean code/OOP principles

**Minor Updates:**
- Updated references in `main.qml`, `README.md`, and prediction GUI files
- Adjusted imports to reflect renamed files

This merge implements issue #494 (clean code OOP breakdown) by separating backend logic into dedicated API modules, making the codebase more maintainable and modular.
